force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -55,10 +64,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return The maximum of the two numbers, or null if either number is null.
      */
     public static Integer findMax(Integer num1, Integer num2) {
-        // Initialize largestNum as null
-        Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if (num1 == null || num2 == null) {
+            return null;
+        } else {
+            if (num1 == num2) {
+                return num1;
```

Comment:
Using the Math.max() method will return num1 or num2 so you don't need to return num1 here.